srcpkgs/wine/template

```diff
@@ -52,6 +56,10 @@ if [ "$XBPS_LIBC" = "glibc" ]; then
 	hostmakedepends+=" prelink"
 fi
 
+if [ "${build_option_mingw}" ]; then
+	hostmakedepends+=" cross-${XBPS_TARGET_MACHINE}-w64-mingw32"
```

Comment:
Just add an entry to the hostmakedepends definition using `vopt_if`, it's cleaner.

Also, you need to strip `-musl` from the target machine, otherwise the build fails.